src/main/scala/AssemblyHelpers.scala

```diff
@@ -0,0 +1,16 @@
+package net.bzzt.reproduciblebuilds
+
+import sbtassembly.{Assembly, AssemblyPlugin}
+import sbtassembly.AssemblyPlugin.autoImport.{Assembly => _, baseAssemblySettings => _, _}
+import sbt._
+import sbt.Keys._
+
+object AssemblyHelpers {
+  val plugin: Plugins.Basic = AssemblyPlugin
+
+  val settings: Seq[Setting[_]] =
+    Seq(
+      assembly := ReproducibleBuildsPlugin.postProcessJar(assembly.value)
```

Comment:
@raboof I'm having a hard time figuring out what key should be used here, any idea?
It is not processing or producing the stripped file.